app/common/renderer/utils/source-parsing.js

```diff
@@ -1,8 +1,11 @@
 import {DOMParser, XMLSerializer} from '@xmldom/xmldom';
 import _ from 'lodash';
 
-export const domParser = new DOMParser();
-export const xmlSerializer = new XMLSerializer();
+const domParser = new DOMParser();
+const xmlSerializer = new XMLSerializer();
+
+export const xmlToDOM = (string) => domParser.parseFromString(string, 'application/xml');
```

Comment:
the new version needs to use MIME_TYPE enum, which is defined in the module. See https://github.com/appium/appium/pull/20513/commits/85c01a679f2af1b5b6a9df5b68f2586cb88801b9

Reply:
I wonder if the issue in the Appium repo was caused by the build tool. Here, tests ran fine on 0.9.0 even without the enum.

Reply:
But it is probably safer to use the built-in enums anyway

Reply:
I'm sure this is because inspector source don't have typescript checks enabled (I would really like they had)

Reply:
Changed to use the enum, and also updated the dependency itself (might as well)